plugins/csp-simple-objects/gui/js/csp-simple-objects-editor.js

```diff
@@ -0,0 +1,759 @@
+
+                div.onkeypress = (e) => {
+                    if (e.keyCode == 9) {
+                        e.preventDefault();
+                        // let nextElement = null;
+                        // if(e.shiftKey) {
+                        //     nextElement = this.tabRevMap.get(document.activeElement)
+                        // } else {
+                        let nextElement = this._tabFwdMap.get(document.activeElement)      
+                        //}
+                        if (nextElement != null) {
+                            nextElement.focus();
+                        }
+                    }
+                };
+            }
+
+            initTabCallback(this._nameDiv, this._rotationWDiv, this._anchorDiv);
+            initTabCallback(this._anchorDiv, this._nameDiv, this._locationLngDiv);
+            initTabCallback(this._locationLngDiv, this._anchorDiv, this._locationLatDiv);
+            initTabCallback(this._locationLatDiv, this._locationLngDiv, this._elevationDiv);
+            initTabCallback(this._elevationDiv, this._locationLatDiv, this._scaleDiv);
+            initTabCallback(this._scaleDiv, this._elevationDiv, this._sizeDiv);
+            initTabCallback(this._sizeDiv, this._scaleDiv, this._rotationXDiv);
+            initTabCallback(this._rotationXDiv, this._scaleDiv, this._rotationYDiv);
+            initTabCallback(this._rotationYDiv, this._rotationXDiv, this._rotationZDiv);
+            initTabCallback(this._rotationZDiv, this._rotationYDiv, this._rotationWDiv);
+            initTabCallback(this._rotationWDiv, this._rotationZDiv, this._nameDiv);
+
+            document.querySelector("#simple-objects-editor-anchor + div > button").onclick = () => {
+                this._anchorDiv.value = CosmoScout.state.activePlanetCenter;
+            };
+
+            document.querySelector("#simple-objects-editor-rotation-w + div > button").onclick = () => {
+                this._rotationXDiv.value = CosmoScout.state.observerRotation[0];
+                this._rotationYDiv.value = CosmoScout.state.observerRotation[1];
+                this._rotationZDiv.value = CosmoScout.state.observerRotation[2];
+                this._rotationWDiv.value = CosmoScout.state.observerRotation[3];
+            };
+            
+            document.querySelector("#simple-objects-editor-pick-location").onclick = () => {
+                this.setPickLocationEnabled(!this._pickLocationButtonActive);
+            };
+
+        // close button -------------------------------------------------------------------------------
+            document.querySelector("#simple-objects-editor-close").onclick = () => {
+                if(this._nameDiv.value != null) {
+                    CosmoScout.callbacks.simpleObjects.undoEdit(this._nameDiv.value);
+                }
+                this._closeEditor();
+            };
+
+        // Delete bookmarks on delete button click -----------------------------------------------------
```

Comment:
```suggestion
        // Delete model on delete button click -----------------------------------------------------
```

I think you should look through the code and replace all occurrences of "bookmark".